php/class-wp-cli.php

```diff
@@ -21,28 +67,149 @@ public function create_guest_authors( $args, $assoc_args ) {
 		global $coauthors_plus;
 
 		$defaults = array(
-				// There are no arguments at this time
-			);
+			'roles'             => array( 'administrator', 'editor', 'author', 'contributor' ),
+			'batch-size'        => 1000,
+			'force-subscribers' => false,
+			'not-a-dry-run'     => false,
+			'log-output'        => false,
+		);
+
 		$this->args = wp_parse_args( $assoc_args, $defaults );
 
-		$users = get_users();
-		$created = 0;
-		$skipped = 0;
-		$progress = \WP_CLI\Utils\make_progress_bar( 'Processing guest authors...', count ( $users ) );
-		foreach ( $users as $user ) {
+		if ( is_array( $this->args['roles'] ) ) {
+			$role_whitelist = $this->args['roles'];
+		} else {
+			$role_whitelist = explode( ',', $this->args['roles'] );
+		}
```

Comment:
This is really just minor and can be ignored, but I still believe worth pointing out. Cleaner:
```
$role_whitelist = $this->args['roles'];
if ( false === is_array( $role_whitelist ) ) {
	$role_whitelist = explode( ',', $role_whitelist );
}
```

Reply:
Done :+1: